src/main/java/edu/wpi/first/smartdashboard/SmartDashboard.java

```diff
@@ -1,6 +1,6 @@
 package edu.wpi.first.smartdashboard;
 
-import edu.wpi.first.smartdashboard.extensions.FileSniffer;
+//import edu.wpi.first.smartdashboard.extensions.FileSniffer;
```

Comment:
Probably better to remove this import instead of commenting it out.

Reply:
Fixed